# Generated by Django 5.1.2 on 2024-11-10 11:31

import django.contrib.auth.validators
from django.db import migrations, models


class Migration(migrations.Migration):

    dependencies = [
        ("users", "0002_customer_meli_code_seller_meli_code"),
    ]

    operations = [
        migrations.RemoveField(
            model_name="myuser",
            name="user",
        ),
        migrations.AlterField(
            model_name="myuser",
            name="otp",
            field=models.CharField(blank=True, max_length=64, null=True),
        ),
        migrations.AlterField(
            model_name="myuser",
            name="username",
            field=models.CharField(
                error_messages={"unique": "A user with that username already exists."},
                help_text="Required. 150 characters or fewer. Letters, digits and @/./+/-/_ only.",
                max_length=150,
                unique=True,
                validators=[django.contrib.auth.validators.UnicodeUsernameValidator()],
                verbose_name="username",
            ),
        ),
    ]
